src/components/RoomsList/RoomsListStyled.js

```diff
@@ -1,6 +1,26 @@
 import styled from 'styled-components';
 import { device } from '@/constants/mediaQueries';
 
+export const UnreadMessagesCount = styled.div`
```

Comment:
Isn't should be span or p tag instead of div?

Reply:
done